### PR TITLE
add sample.env and update docs with new env vars

### LINKED ISSUE #306: contributing.MD is missing info on required env variables

### PR DESCRIPTION
**Pull request by mikejoh12**
## What issue is this solving?

Closes #306

### Description
Adds a sample.env file and instructions on how to obtain the environmental variables. Also fixes a broken link in contributing.MD.

## Any helpful knowledge/context for the reviewer?

- Is a re-seeding of the database necessary? No
- Any new dependencies to install? No
- Any special requirements to test? No
  - (e.g., admin perms, alt account, etc.)

### Please make sure you've attempted to meet the following coding standards
- [X] Code has been tested and does not produce errors
- [X] Code is readable and formatted
- [X] There isn't any unnecessary commented-out code

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,8 +39,10 @@ $ npm install
 
 ### Add Secret Keys
 
-1. In the cloned repo, create a file named `.env`. This will store environment variables you may want to keep secret.
-2. Create a variable named `DISCORD_SECRET_KEY` (you will assign this a value later on).
+1. The cloned repo includes a file named sample.env which contain the environmental variables. Copy this file to another file named .env by running the command below in the project main directory. Several of these variables need to be kept secret so this file is not commited to github.
```
Comment (aedwardg) [inline CONTRIBUTING.md:42]: A few minor nit-picks:

- "sample.env which contain" ➡️  "sample.env which contains"
- "so this file is not commited to github" ➡️ "so the `.env` file is not committed to GitHub"

**Review by aedwardg — COMMENTED** [review]
Looks great, but I think it needs a sentence or two about running the `deployCommands` script to register the slash commands or else they won't work.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mikejoh12) [reply CONTRIBUTING.md]: I made those changes and added a script to package.json which deploys the application commands.

**Review by aedwardg — APPROVED** [review]
👍

**Review by lyallstewart — APPROVED** [review]
Looks great!

**Pull request merged**